src/screens/SendTokens.js

```diff
@@ -136,6 +136,9 @@ class SendTokens extends React.Component {
       const instance = ref.current;
       const dataOne = instance.getData();
       if (!dataOne) return;
+      if (hathorLib.wallet.isHardwareWallet()) {
+        dataOne = instance.handleInitialData(dataOne);
```

Comment:
Why don't we do it always? I mean, why only for hardware wallets?

Reply:
The software wallet uses another method called `this.props.wallet.sendManyOutputsTransaction`